blog/_posts/2019-08-01-commit-of-the-month-july.md

```diff
@@ -0,0 +1,63 @@
+---
+slug: july-tilt-commit-of-the-month
+date: 2019-08-01
+author: maria
+layout: blog
+title: "Tilt Commit of the Month: July 2019"
+subtitle: "No Browser - Flag to Disable the Web UI"
+images:
+  - featuredImage.png
+image_type: "contain"
+tags:
+  - docker
+  - kubernetes
+  - microservices
+  - tilt
+  - cotm
+keywords:
+  - web-interface
+  - terminal
+  - tilt
+  - flag 
+---
+
+
+Welcome the second edition of Commit of the Month, a series of blog posts where we highlight some of the work done on Tilt this month! 
+
+This month’s Commit of the Month is the command line flag that disables the webUI:
+
+* [web: flag for opening browser](https://github.com/windmilleng/tilt/pull/1830)
+
+## What does it do?
+
+
+When you Tilt Up, usually you will see two UIs - the one that shows up on your terminal and the webUI.
+
+This one pops up on your _browser_..
+![Web UI](/assets/images/july-tilt-commit-of-the-month/webUI.png) 
+
+And this one pops up on your _terminal_.. 
+![Terminal UI](/assets/images/july-tilt-commit-of-the-month/TUI.png)
+
+Look familiar? 
+
+Well now, if you want to disable the web UI, you can now tilt up along with “--no-browser” 
```

Comment:
same nit as before -- you're not disabling it, just making it not pop up. (i think the most common use of this is probably people who already have the web UI up in a tab and don't want tilt to pop up ANOTHER one)

Reply:
Yes! Thank you this is a really good point